.github/workflows/backend.yml

```diff
@@ -24,6 +24,8 @@ on:
       - dev
     paths-ignore:
       - 'docs/**'
+      - 'README.md'
+      - 'README_zh_CN.md'
```

Comment:
Ignore all markdown files would be better
```suggestion
      - '**/*.md'
```

Reply:
SGTM, addressed